templates/account/invoices/index.html

```diff
@@ -80,6 +80,31 @@ <h1>Invoices</h1>
             {% endfor %}
           </tbody>
         </table>
+        <nav class="p-pagination" aria-label="Pagination">
```

Comment:
I forgot to tell you before, but I remembered while reviewing that we have a partial for navigation already.
If you replace your changes with the following block, navigation should still work and will also be consistent with the rest of the navigations in the site.

```
  <section class="p-strip is-shallow">
    {% with %}
      {% set total_pages = total_pages %}
      {% set current_page = current_page %}
      {% include "shared/_pagination.html" %}
    {% endwith %}
  </section>
```